content/blog/online-privacy.en.md

```diff
@@ -3,43 +3,49 @@
 
 Privacy Badger is available for Chrome, Opera, Firefox, and even Firefox on Android phones.
 
 ## uBlock Origin
 
-[uBlock Origin](https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm?hl=en) is one of the most popular ad blocker extensions. It prevents pesky popups and widgets from sending your data to advertisers. uBlock Origin lets you create personalized lists which gives you more control over what ads to block and what ads to allow. It has also been found to effectively block more ads than others while using less memory, so it doesn’t slow down your browsing experience.
+<img src="/img/blog/online-privacy/ublock-origin.png">
 
-Unlike ad blockers, anti-tracking extensions block cookies and protect your online anonymity by preventing tracking and malvertising of third-party advertisers.
+[uBlock Origin](https://ublock.org/) is one of the most popular ad and content blocker extensions. It prevents pesky popups and widgets from sending your data to advertisers. uBlock Origin lets you create personalized lists which gives you more control over what ads to block and what ads to allow. It has also been found to effectively block more ads than others while using less memory, so it doesn’t slow down your browsing experience.
 
 uBlock Origin is available for Chrome, Opera, Firefox, and Safari.
 
-## Ghostery
-
-[Ghostery](https://www.ghostery.com/) is similar to Privacy Badger, but with a slightly better interface. It detects cookies on websites you visit and blocks the tracking to speedup page-load times—essentially eliminating clutter while protecting your data. However, Ghostery doesn’t block trackers by default. You’ll need to inspect them from the interface and decide what to do with them. There’s also an option to share your data with Ghostery to improve your experience.
-
 ## HTTPS Everywhere
 
-Secure websites use the HTTPS (Hyper Text Transfer Protocol Secure) protocol, which encrypts your credit card details, personal information, and other sensitive data. However, not all websites are HTTPS-enabled, so the next step and piece-de-resistance in your privacy setup should be a good site-securing extension.
+<img src="/img/blog/online-privacy/https.png">
+
+Secure websites use the HTTPS (Hyper Text Transfer Protocol Secure) protocol, which encrypts your credit card details, personal information, and other sensitive data. However, not all websites are HTTPS-enabled, so it redirects you to the HTTPS version of the site if it has one.
```

Comment:
You didn't mention the extension yet, so you can't refer to it as `it` here.

Reply:
Oh yeah will take care